src/components/states/emptyState/emptyState.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { IconType } from '../../icon';
+import { EmptyState } from './emptyState';
+
+const meta: Meta<typeof EmptyState> = {
+    title: 'components/States/EmptyState',
+    component: EmptyState,
+    tags: ['autodocs'],
+    parameters: {
+        design: {
+            type: 'figma',
+            url: 'https://www.figma.com/file/ISSDryshtEpB7SUSdNqAcw/branch/jfKRr1V9evJUp1uBeyP3Zz/Aragon-ODS?type=design&node-id=10095%3A21633&mode=dev&t=FtMO7nBXAzYBFGaW-1',
+        },
+    },
+};
+
+type Story = StoryObj<typeof EmptyState>;
+
+/**
+ * Default EmptyState component with minimum props.
+ */
+export const Default: Story = {
+    args: {
+        heading: 'Heading',
+        description: 'Description',
+        objectIllustration: { object: 'LIGHTBULB' },
+    },
+};
+
+/**
+ * Stacked EmptyState component with full props examples for Object Illustration.
+ */
+export const StackedFullWithObject: Story = {
+    args: {
+        heading: 'Heading',
+        description: 'Description',
+        objectIllustration: { object: 'LIGHTBULB' },
+        primaryButton: {
+            label: 'Label',
+            iconLeft: IconType.ADD,
+            iconRight: IconType.CHEVRON_RIGHT,
+            onClick: () => alert('Primary Button Clicked'),
+        },
+        secondaryButton: {
+            label: 'Label',
+            iconLeft: IconType.ADD,
+            iconRight: IconType.CHEVRON_RIGHT,
+            onClick: () => alert('Secondary Button Clicked'),
+        },
+    },
+    render: (args) => <EmptyState {...args} />,
```

Comment:
The `render` here and on the stories below is not needed, right?
```suggestion
```

Reply:
Misclicked between tabs. fixing now